src/tr1/config.h

```diff
@@ -111,6 +111,10 @@ typedef struct {
         int32_t cntlr_layout;
     } input;
 
+    struct {
+        bool highlight_pickups;
+    } visuals;
```

Comment:
Maybe it should be `gameplay.enable_pickup_aids`?

Reply:
Done

Reply:
Thanks